core/translate/schema.rs

```diff
@@ -699,6 +738,44 @@ pub fn translate_drop_table(
         cursor_id: sqlite_schema_cursor_id_0,
         dest: row_id_reg,
     });
+    if let Some((cdc_cursor_id, _)) = cdc_table {
+        let table_type = program.emit_string8_new_reg("table".to_string()); //  r4
+        program.mark_last_insn_constant();
+
+        let skip_cdc_label = program.allocate_label();
+
+        let entry_type_reg = program.alloc_register();
+        program.emit_column(sqlite_schema_cursor_id_0, 0, entry_type_reg);
+        program.emit_insn(Insn::Ne {
```

Comment:
Could you please comment this the same way you commented it in the PR description?